include/edcwccwi.h

```diff
@@ -8,8 +8,10 @@
 
                    #ifndef __edcwccwi
                    #define __edcwccwi 1
+#if !defined(__clang__) || defined(__ibmxl__)
```

Comment:
Don't add this guard..  Let ibm-clang see these pragmas.

Reply:
ibm-clang ignores those pragmas, as shown if `-Wall` were used:
```
In file included from zos.cc:16:
zoslib/include/edcwccwi.h:12:28: error: unknown pragma ignored [-Werror,-Wunknown-pragmas]
                   #pragma nomargins nosequence
                           ^
zoslib/include/edcwccwi.h:13:28: error: unknown pragma ignored [-Werror,-Wunknown-pragmas]
                   #pragma checkout(suspend)
                           ^
zoslib/include/edcwccwi.h:617:28: error: unknown pragma ignored [-Werror,-Wunknown-pragmas]
                   #pragma checkout(resume)
```
At least [GN](https://gn.googlesource.com/gn/), which depends on zoslib, builds with -Wall (and -Werror).